### ISSUE TITLE
Make martinize2 accessible in python

### ISSUE THREAD

**Issue by OLaprevote** [marrink-lab/vermouth-martinize#360]
Couldn't martinize2 exist as a python function too? This way it would be possible to call it directly in python

Ex:
```python
from vermouth import martinize2

input_file = 'my/path/to/inp.pdb'
output_file = 'my/path/to/out.pdb'
topology_file = 'my/path/to/out.top'

martinize2(input_file, output_file, top_path=topology_file, cystein_bridge='auto',
                  neutral_termini=True, to_ff='martini3001', posres='backbone')
```

Everything is already written in `bin/martinize2` and it feels like a loss not to give the choice. I was writing a jupyter notebook and I thought that having it as a python function would be more practical than calling the bash command. Not too hard to implement either: I would gladly spend the time to do these changes.

**Comment by pckroon** [comment]
I strongly agree, and I think it would be a great idea. I would be very open to a PR, if you have the time. I'm currently rather swamped unfortunately, but review shouldn't be too hard.
